CONTRIBUTING.md

```diff
@@ -43,7 +42,14 @@ Once you have built your template file or files, the commit message should have
 
 * for template modifications without a version change, use ```<pkgname>: <reason>``` ([example](https://github.com/void-linux/void-packages/commit/8b68d6bf1eb997cd5e7c095acd040e2c5379c91d)).
 
+* for package removals, use ```<pkgname>: remove package``` ([example](https://github.com/void-linux/void-packages/commit/83784632d94deee5d038c8e1c4c1dffa922fca21)).
+
+* for `common/shlibs` modifications, use `common/shlibs: <pkgname>` ([example](https://github.com/void-linux/void-packages/commit/613651c91811cb4fd2e1a6be701c87072d759a9f)).
+
 If you want to describe your changes in more detail, add an empty line followed by those details ([example](https://github.com/void-linux/void-packages/commit/f1c45a502086ba1952f23ace9084a870ce437bc6)).
+Pull requests with changes to packages that take too long to build, such as Firefox and the Linux kernel, should include `[ci skip]` in the last line of the commits, to avoid unnecessarily
+spending the CI builders' time, since the package is not going to finish building under the 50 minute limit.
+In such cases, the package should be built and cross-built locally for a combination of architectures and libc.
```

Comment:
Maybe something like:
> Pull requests are automatically submitted for Continuous Integration (CI) testing to ensure packages build on various architectures and C libraries. Packages that take longer than 50 minutes to build (for example, Firefox or the Linux kernel) will fail CI and should include `[ci skip]` in the commit message of the last commit in the PR to avoid wasting CI builder time. Use your best judgment on build times based on your local building experience. If you skip CI when submitting a PR, please locally build for a variety of architectures with glibc as well as musl and note your local results in PR comments.

Reply:
That is a bit better. I prefer "libc" rather than "C library", though.

Reply:
I don't object to "libc" instead of "C libraries", except it's harder to pluralize. :)

Reply:
Added some more changes.